hw/ip/i2c/dv/i2c_sim_cfg.hjson

```diff
@@ -59,16 +59,19 @@
     {
       name: i2c_fifo_watermark
       uvm_test_seq: i2c_fifo_watermark_vseq
+      reseed: 30
```

Comment:
any reason to reduce the seed number?

Reply:
@weicaiyang The FIFO depth is double to 64 so some FIFO test can run slow